configs/node.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  env: {
+    node: true
+  },
+  extends: [
+    'plugin:bpmn-io/recommended'
+  ],
+  parserOptions: {
+    ecmaVersion: 'latest'
```

Comment:
One oversight: Shouldn't we pin this to ES2018, too?

Otherwise we _may_ slip in ES2022+ features into our node-style (but also browser supported) utilities.

Reply:
My take is that we want to enforce ES2018 for the browsers, and this is due to browsers support. In Node, we don't have this limitation. We have two options:

1. Every Node project has to set `parserOptions#ecmaVersion` in order to use ES2019+
2. Node-style but browser supported utilities have to specify `parserOptions#ecmaVersion`

Until now, we have accepted any syntax due to babel parser. After this PR was merged, we are enforcing option 2. I'd say let's try this out, and we can go even more strict if we see we need to.

Reply:
So you could argue what is missing is a vanilla `bpmn-io/recommended` profile (no `node` or `browser` env).

Reply:
Done via https://github.com/bpmn-io/eslint-plugin-bpmn-io/pull/17